src/mcnp/tests/dagmcnp_unit_tests.cpp

```diff
@@ -113,7 +113,7 @@ TEST_F(DAGMCNP5Test, dagmcinit_test) {
     EXPECT_EQ(expected_lcad[i], input_deck[i]);
   }
   // delete the lcad file
-  std::remove("lcad");
+  //std::remove("lcad");
```

Comment:
Why this change?  Should we just delete it?  or is this an artifact of confirming the change?

Reply:
Oops. I accidentally left that in when I was looking at the written file. I'll fix that.